cmd/single-ocsp/main.go

```diff
@@ -0,0 +1,169 @@
+// Copyright 2015 ISRG.  All rights reserved
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package main
+
+import (
+	"crypto/x509"
+	"encoding/json"
+	"io/ioutil"
+	"os"
+
+	"github.com/letsencrypt/boulder/cmd"
+
+	"github.com/letsencrypt/boulder/Godeps/_workspace/src/github.com/cloudflare/cfssl/crypto/pkcs11key"
+	"github.com/letsencrypt/boulder/Godeps/_workspace/src/github.com/codegangsta/cli"
+	"github.com/letsencrypt/boulder/Godeps/_workspace/src/golang.org/x/crypto/ocsp"
+)
+
+// PKCS11Config defines how to load a module for an HSM.
+// XXX(rlb@ipv.sx) Copied from certificate-authority.go
+type PKCS11Config struct {
+	Module string
+	Token  string
+	PIN    string
+	Label  string
+}
+
+func readFiles(c *cli.Context) (issuer, responder, target *x509.Certificate, template ocsp.Response, pkcs11 PKCS11Config, err error) {
+	// Isser certificate
+	issuerFileName := c.GlobalString("issuer")
+	issuerBytes, err := ioutil.ReadFile(issuerFileName)
+	if err != nil {
+		return
+	}
+
+	issuer, err = x509.ParseCertificate(issuerBytes)
+	if err != nil {
+		return
+	}
+
+	// Responder certificate
+	responderFileName := c.GlobalString("responder")
+	responderBytes, err := ioutil.ReadFile(responderFileName)
+	if err != nil {
+		return
+	}
+
+	responder, err = x509.ParseCertificate(responderBytes)
+	if err != nil {
+		return
+	}
+
+	// Target certificate
+	targetFileName := c.GlobalString("target")
+	targetBytes, err := ioutil.ReadFile(targetFileName)
+	if err != nil {
+		return
+	}
+
+	target, err = x509.ParseCertificate(targetBytes)
+	if err != nil {
+		return
+	}
+
+	// OCSP template
+	templateFileName := c.GlobalString("template")
+	templateBytes, err := ioutil.ReadFile(templateFileName)
+	if err != nil {
+		return
+	}
+
+	err = json.Unmarshal(templateBytes, &template)
+	if err != nil {
+		return
+	}
+
+	// PKCS#11 config
+	pkcs11FileName := c.GlobalString("pkcs11")
+	pkcs11Bytes, err := ioutil.ReadFile(pkcs11FileName)
+	if err != nil {
+		return
+	}
+
+	err = json.Unmarshal(pkcs11Bytes, &pkcs11)
+	return
+}
+
+func main() {
+	app := cli.NewApp()
```

Comment:
nit: Since you don't use custom `cli.Command`s you can just use `cmd.NewAppShell` and avoid having to manually set the version, author, and email.